src/integrationTest/resources/tests/usermanagement/SecurityGroupControllerGetTest/securityGroupsGetEndpointAllReturned.json

```diff
@@ -0,0 +1,13 @@
+[
+  {"id":-17,"security_role_id":10,"global_access":true,"display_state":true,"courthouse_ids":[],"name":"Mid Tier Group"},
+  {"id":-16,"security_role_id":9,"global_access":true,"display_state":true,"courthouse_ids":[],"name":"Dar Pc Group"},
+  {"id":-15,"security_role_id":8,"global_access":true,"display_state":true,"courthouse_ids":[],"name":"Cpp Group"},
+  {"id":-14,"security_role_id":7,"global_access":true,"display_state":true,"courthouse_ids":[],"name":"Xhibit Group"},
+  {"id":-6,"security_role_id":6,"global_access":true,"display_state":true,"courthouse_ids":[],"name":"Test RCJ Appeals"},
+  {"id":-5,"security_role_id":5,"global_access":false,"display_state":true,"courthouse_ids":[],"name":"Test Language Shop"},
+  {"id":-4,"security_role_id":4,"global_access":false,"display_state":true,"courthouse_ids":[],"name":"Test Transcriber"},
+  {"id":-3,"security_role_id":3,"global_access":false,"display_state":true,"courthouse_ids":[],"name":"Test Judge"},
+  {"id":-2,"security_role_id":2,"global_access":false,"display_state":true,"courthouse_ids":[],"name":"Test Requestor"},
+  {"id":-1,"security_role_id":1,"global_access":false,"display_state":true,"courthouse_ids":[],"name":"Test Approver"},
+  {"id":1,"security_role_id":11,"global_access":true,"display_state":true,"courthouse_ids":[],"name":"ADMIN"}
```

Comment:
Can you check this group please - I'd expect it to be `SECURITY_ADMIN` now after [DMP-2403](https://tools.hmcts.net/jira/browse/DMP-2403)